components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/exception/ExceptionCodes.java

```diff
@@ -71,9 +71,9 @@
     // Workflow related codes
     WORKFLOW_EXCEPTION(900550, "Workflow error", 500,
             "Error occurred while executing workflow task"),
-    WORKFLOW_NOT_FOUND(900551, "Workflow error", 500,
+    WORKFLOW_NOT_FOUND(900551, "Workflow error", 404,
             "Workflow entry cannot be found for the given reference id"),
-    WORKFLOW_COMPLETED(900552, "Workflow error", 500,
+    WORKFLOW_COMPLETED(900552, "Workflow error", 404,
```

Comment:
Shouldn't this be 409 conflict like the below? Better to change the key to "WORKFLOW_ALREADY_COMPLETED".